### PR TITLE
Add the LRR conformance suite.

### PR DESCRIPTION
**Pull request by stevenbronson-wk**
#### Steps to Test
CI

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: scripts/conformance_suites/run_lrr_conformance_suite.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+from arelle.WebCache import WebCache
+
+# from https://specifications.xbrl.org/work-product-index-registries-lrr-1.0.html
+CONFORMANCE_SUITE = 'tests/resources/conformance_suites/lrr-conf-pwd-2005-06-21.zip/'
+
+ARGS = [
+    '--file', os.path.join(CONFORMANCE_SUITE, 'lrr', 'conf', 'index.xml'),
+    '--testcaseResultsCaptureWarnings',
+    '--validate',
+
```
Comment (austinmatherne-wk) [inline scripts/conformance_suites/run_lrr_conformance_suite.py:14]: nit: extra line

**Review by derekgengenbacher-wf — APPROVED** [review]

File: scripts/conformance_suites/run_lrr_conformance_suite.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import os.path
+from unittest.mock import patch
+
+from arelle.CntlrCmdLine import parseAndRun
+from arelle.WebCache import WebCache
+
+# from https://specifications.xbrl.org/work-product-index-registries-lrr-1.0.html
+CONFORMANCE_SUITE = 'tests/resources/conformance_suites/lrr-conf-pwd-2005-06-21.zip/'
```
Comment (austinmatherne-wk) [inline scripts/conformance_suites/run_lrr_conformance_suite.py:8]: nit: trailing slash initially looked like a mistake. Remove it here and append it in the replace?

**Review by austinmatherne-wk — APPROVED** [review]

**Review by austinmatherne-wk — APPROVED** [review]

File: tests/integration_tests/validation/XBRL/test_lrr_conformance_suite.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+]
+
+if os.getenv('CONFORMANCE_SUITES_TEST_MODE') == 'OFFLINE':
+    ARGS.extend(['--internetConnectivity', 'offline'])
+
+
+oldNormalizeUrl = WebCache.normalizeUrl
+
+
+def normalizeUrl(self, url, base=None):
```
Comment (aaroncameron-wk) [inline tests/integration_tests/validation/XBRL/test_lrr_conformance_suite.py:26]: Is this needed because the test suite is using local file system paths?

**Review by aaroncameron-wk — APPROVED** [review]

**Pull request merged**